pkg/model/components/etcdmanager/model.go

```diff
@@ -315,17 +333,13 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster *kops.EtcdClusterSpec) (*v1.Po
 		BackupStore:   backupStore,
 		GrpcPort:      grpcPort,
 		DNSSuffix:     dnsInternalSuffix,
-		EtcdInsecure:  !isTLS,
+		EtcdInsecure:  etcdInsecure,
 	}
 
-	config.LogVerbosity = 8
+	config.LogVerbosity = 6
 
 	{
-		// @check if we are using TLS
-		scheme := "http"
-		if isTLS {
-			scheme = "https"
-		}
```

Comment:
Feels like we block the option to disable tls as something fixed here? Not that insecure is a good thing but this feels like taking away customizability that was once there and some clusters might rely on.

Reply:
So if we specify etcdInsecure (above), etcd-manager will override the scheme in the URLs here.

We _could_ allow users to specify etcd insecure (somehow).  Currently we default the fields to on when etcd-manager is in use, and because they're `bool` and not `*bool` it is difficult to know whether the value is false or just unset, but we could create another field.

The workaround for now is to set the etcd provider back to legacy, but that option will go away in 1.13.  But if there are people that need insecure, maybe we can rely on this workaround for now.

Reply:
Oh, I missed this is etcdmanager only. Ignore this 👍